.github/workflows/image_build.yml

```diff
@@ -274,6 +278,33 @@ jobs:
         run: |
           echo "Image is available at ttl.sh/localai-ci-pr-${{ github.event.number }}:${{ steps.meta_pull_request.outputs.version }}" >> $GITHUB_STEP_SUMMARY
 ## End testing image
+      - name: Build and push CUDA 11
```

Comment:
We already push cuda-11 images as part of the build matrix here https://github.com/mudler/LocalAI/blob/06de54203206577fb8cfa779be11ba880d7430c0/.github/workflows/image.yml#L65 . I'm not sure if there is anything to add here, except changing image's defaults.

Reply:
Good catch, I'll remove it.

Reply:
Makefile `docker-aio-all:` looks strange, a duplicated command, should that be removed or `DOCKER_AIO_SIZE` changed in the second call?

Reply:
Good catch - that looks a leftover, should be dropped as its not used anymore! Thanks!